test/purdy.js

```diff
@@ -126,7 +138,12 @@ describe('Purdy', function () {
             var mises = function () { this.moop = 3 }
             var obj = { instance: new mises() };
             var out = Purdy.stringify(obj, { indent: 1 });
-            expect(out).to.equal('{\n \u001b[1m\u001b[37minstance\u001b[39m\u001b[22m: {\n  \u001b[1m\u001b[37mmoop\u001b[39m\u001b[22m: \u001b[1m\u001b[34m3\u001b[39m\u001b[22m\n }\n}');
+            var expected = [
+                '{\n \u001b[1m\u001b[37minstance\u001b[39m\u001b[22m: ',
+                funcNameInfer ? '\u001b[32mmises\u001b[39m ' : '',
+                '{\n  \u001b[1m\u001b[37mmoop\u001b[39m\u001b[22m: \u001b[1m\u001b[34m3\u001b[39m\u001b[22m\n }\n}'
+            ].join('');
```

Comment:
I've been thinking Is there any way we can do this in a cleaner way?

For instance:
``` javascript
var inferred = funcNameInfer ? '\u001b[32mmises\u001b[39m ' : '',
var expected = '{\n \u001b[1m\u001b[37minstance\u001b[39m\u001b[22m: ' + inferred + '{\n  \u001b[1m\u001b[37mmoop\u001b[39m\u001b[22m: \u001b[1m\u001b[34m3\u001b[39m\u001b[22m\n }\n}'